### ISSUE TITLE
first-join giving items to everyone, even if they've already been on server

### ISSUE THREAD

**Issue by phat-male** [RockinChaos/ItemJoin#200]
1.) 5.0.2 also tried on 5.0.1 versions.
2.) 1.13.2 spigot
3.) I'm trying to get so people are given armor & a bed on first join to the server. So I've set up a permission flag which now works, cause without it it never seemed to work even if I used the respawn flag. Now however whenever people connect to server that have been several times previously are given the items again if there is nothing in those designated locations.

config file Yaml https://pastebin.com/1SVwbmCL
items file Yaml https://pastebin.com/gRLZN2Fq

**Comment by RockinChaos** [comment]
Thanks for opening a report, the issue is that First-Join is not a trigger. It is an itemflag. Switch it to look like itemflags: first-join and it will work.

**Comment by phat-male** [comment]
Awesome, thanks @RockinChaos for the support especially the very quick response.
Tested it out and that seems to have done the trick. I guess I just must have gotten confused alil as the original generation of the items.yml has "triggers: join, respawn, world-change".
Although I have browsed the https://www.spigotmc.org/wiki/learning-the-items-yml/#defining-an-items-itemflags and keep it saved for reference.

Again, thanks for the help. Great plugin, and wonderful support!!
